utils/mdxUtils.ts

```diff
@@ -47,10 +47,15 @@ export const getDocsOrderList = (basePath: string): DocsOrderList => {
         href,
         title: data.title || slug,
         order: data.order || 0,
+        phase: data.phase || '',
         subMenu: [],
       };
     })
-    .sort((a, b) => a.order - b.order);
+    .sort((a, b) => a.order - b.order)
+    .filter((list) => {
+      if (process.env.NODE_ENV === 'development') return true;
+      return list.phase !== 'develop';
```

Comment:
Is there a reason to use a separate short name? I think I'll get confused later about whether `develop` or `development`.

```suggestion
      return list.phase !== 'development';
```

Reply:
I agree. I've changed the name.